src/components/calculator.js

```diff
@@ -0,0 +1,113 @@
+import { Component } from 'react';
+import './calculator.css';
+
+/* eslint-disable react/prefer-stateless-function */
+class Calculator extends Component {
```

Comment:
- [ ] Linters help us write clean code. Let us maintain that. Kindly enable linters here.

  **Suggestion**: For the linter error, as suggested in the linter message, you could set up your state within a constructor, before the `render` method, like so:

    ```
      constructor(props) {
          super(props);
          this.state = {};
      }
    ```